tutorials/beginner/installing.md

```diff
@@ -1,27 +1,55 @@
 ---
 layout: tutorial-beginner
+title: Installing Extension
 ---
 
 {% include base.html %}
 
-# To be done
+# Installing an Extension
```

Comment:
Indeed in ESH the term "extension" is used, in OH instead "add-on". I would keep the wording as "Add-ons" here.
Maybe we should check if the label of the Paper UI can be overwritten in OH to show "Add-ons" in the menu instead (so please do not include too many screenshots in the docs yet, as such changes will incur quite some work wrt updating the docs...)